workflows/theiameta/wf_theiameta_illumina_pe.wdl

```diff
@@ -60,18 +62,39 @@ workflow theiameta_illumina_pe {
       kraken2_report = kraken2_clean.kraken2_report,
       samplename = samplename
   }
-  call metaspades_assembly_wf.metaspades_assembly_pe as metaspades {
+  call metaspades_task.metaspades_pe {
     input:
-      read1 = read_QC_trim.read1_clean,
-      read2 = read_QC_trim.read2_clean,
+      read1_cleaned = read1,
```

Comment:
Should these task inputs be set to read_QC_trim outputs? I.e. 
```
  call metaspades_task.metaspades_pe {
    input:
      read1 = read_QC_trim.read1_clean,
      read2 = read_QC_trim.read2_clean,
      samplename = samplename
    }
```

Reply:
Oh my god! Thank you!